Makefile

```diff
@@ -120,6 +120,9 @@ circleci-container:			#: Build the linux container for CircleCi
 check-container:
 	$(MAKE) linux-container CONTAINER_NAME=check
 
+torcharrow-container:
```

Comment:
strictly speaking it's a "manylinux-builder" container. For example in PyTorch's manylinux docker builder is called `pytorch/manylinux-builder:cpu` or `cpu-main`: https://hub.docker.com/layers/pytorch/manylinux-builder/cpu-main/images/sha256-1d8d59abea78ea6edeed1a1ecd0e3705b2df24e006c5e56cbc7bc6b398adb17b?context=explore

But I definitely won't object calling it `velox-torcharrow` container :)

Reply:
While this is correct, most velox devs arent aware of manylinux and would wonder what thats for, I can rename it to velox-torcharrow.

Reply:
@kgpai : Sorry, i mean i don't object the current name (e.g. velox-torcharrow or torcharrow container)